### PR TITLE
Fix BIC doc

### PR DESCRIPTION
**Pull request by mbaudin47**
Fixes the description of the KS test so that it matches the implementation.

### CODE REVIEW

**Comment by mbaudin47** [comment]
I think I'm wrong: I created a single PR with two different modifications. Don't know how to separated things yet.

File: python/doc/theory/data_analysis/kolmogorov_test.rst [hunk 1]
```diff
@@ -33,28 +33,43 @@ With a sample :math:`\left\{ x_1,x_2,\ldots,x_N \right\}`, the distance is estim
...
+Two situations may occur in practice.
+
+- The parameters of the distribution under test are known. 
+In this case, algorithms are available to directly compute 
+the distribution of :math:`\widehat{D}_N` both for *N* large 
+(asymptotic distribution) or for *N* finite. 
+This is because the distribution of :math:`\widehat{D}_N` does 
+not depend on the candidate distribution. 
+- The parameters of the distribution under test are estimated 
+from data.
```
Comment (jschueller) [inline python/doc/theory/data_analysis/kolmogorov_test.rst]: need a blank line after the bullet list

**Comment by jschueller** [comment]
You can drop the first commit with an interactive rebase.

**Comment by mbaudin47** [comment]
The KS doc is managed in PR #1032. This commit is for BIC doc.
I have to manage git in order to do so.

**Comment by mbaudin47** [comment]
No the BIC doc is managed in PR #1021. This is why I close this unnecessary PR.

**Pull request closed without merging**